src/tasks/AppleAppBuilder/Templates/runtime.m

```diff
@@ -375,8 +375,17 @@ static bool is_pinvoke_override_library (const char* libraryName)
     mono_set_crash_chaining (TRUE);
 
     if (wait_for_debugger) {
-        char* options[] = { "--debugger-agent=transport=dt_socket,server=y,address=0.0.0.0:55556" };
-        mono_jit_parse_options (1, options);
+        argc++;
+        char** copy_argv = (char**) malloc (argc * sizeof (char*));
+        memcpy (copy_argv, argv, (argc - 1) * sizeof (char*));
+        copy_argv [argc - 1] = strdup ("--debugger-agent=transport=dt_socket,server=y,address=0.0.0.0:55556");
```

Comment:
nit: I think it is a bit more clear when the arg count increment is done after copying, eg:
```c
        char** argv_ext = (char**) malloc ((argc + 1) * sizeof (char*));
        // copy passed args
        memcpy (argv_ext, argv, argc * sizeof (char*));
        // add an extra arg
        argv_ext [argc] = strdup ("--debugger-agent=transport=dt_socket,server=y,address=0.0.0.0:55556");
        argc++;
```